### ISSUE TITLE
iCloud doesn’t load

### ISSUE THREAD

**Issue by broyuken** [gcobb321/icloud3#43]
I installed icloud3 on 0.111.1 via hacs and restarted. No errors, no logs saying it even tried to load. Tried multiple times. I have installed it in the past fine but for some reason now it’s just not even attempting to load. All the files are in custom components like they should be.

**Comment by gcobb321** [comment]
I just did a 0.111.1 update and it loads fine on my system. I didn’t do a hacs update, just did an HA ... upgrade on the supervisor sidebar. The ha updateDetails  said something about starting ha before all Integrations has loaded. Could that have something to do with your issue? I don’t know.

I am running the v2.2.0 release candidate. I’m updating rc5 to rc6 tomorrow. It’s in the development-v2.2.0 rc6 Directory on the iCloud3 GitHub repository.

What happens if you do a ha restart? Also make sure the configuration is set up. Maybe they changed something. Did you see anything in the HA log file? add log_level: debug and see iCloud3 loads with more info that can tell what is going on.

————————
Gary Cobb

**Comment by broyuken** [comment]
I had a moment. I recently rebuilt my HA to resolve some instability issues at the same time that that iOS app was removed from the store. To stop a bunch of errors in my logs I temporarily commented out my icloud3 config and never re-enabled it. Totally forgot about it until you said that. I in-commented it and now it’s loaded. Sorry to waste your time with a stupid issue. Had a bad day and jumped the gun on opening the issue before investigating further.
